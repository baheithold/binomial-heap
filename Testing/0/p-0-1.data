3534 5534 402 ; 5102 584 374 ; 5455 9619 84 ; 2717 5266 139 ; 4019 1283 344 ; 
3001 1807 11 ; 6473 4183 436 ; 6658 7874 201 ; 7933 1502 42 ; 1819 2794 433 ; 

