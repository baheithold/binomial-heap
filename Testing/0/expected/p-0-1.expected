0: 9397
1: 9494(9397)423
weight: 423
